storage/database/dynamodb_test.go

```diff
@@ -21,13 +21,33 @@
 package database
 
 import (
+	"io"
+	"os"
 	"testing"
 
+	"github.com/klaytn/klaytn/log"
+	"github.com/klaytn/klaytn/log/term"
+	"github.com/mattn/go-colorable"
+
 	"github.com/klaytn/klaytn/common"
 	"github.com/klaytn/klaytn/common/hexutil"
 	"github.com/stretchr/testify/assert"
 )
 
+func enableLog() {
```

Comment:
Do we need this here?

Reply:
I thought it would be good to call `enableLog()`  in `testDynamoBatch_WriteMutliTables` since it ends a test with Crit when it fails.
Do you think it is not a good idea to put here?

Reply:
No, I just wondered as we don't usually check logs for testing. Just wanted to know when it is needed.